examples/text_classification_with_scriptable_tokenzier/README.md

```diff
@@ -0,0 +1,50 @@
+# Text Classfication using a Scriptable Tokenizer 
+
+This example shows how to combine a text classification model with a scriptable tokenizer into a single artifact and deploy it with TorchServe.
+The combination of model and tokenizer into a single artifact reduces the handler complexity and ensures that the same tokenizer is used during training and inference 
```

Comment:
>  into a single artifact reduces the handler complexity

are you referring to not needing to pass a tokenizer in extra files? 

> ensures that the same tokenizer is used during training and inference 

Could you elaborate a bit more on this, I'm not familiar with this general problem